config/build/dpkg_build/control

```diff
@@ -3,7 +3,7 @@ Version: 0.5.1.1
 Section: base
 Priority: optional
 Architecture: amd64
-Depends:
+Depends: libmariadb-dev-compat | libmariadb-dev | libmariadbclient18 | libmariadb3
 Maintainer: Thomas Pollerspöck (Thomas.Pollerspoeck@de.bosch.com)
 Description: Robot Framework AIO (All In One)
```

Comment:
RobotFramework AIO 

no space between Robot and Framework.

Reply:
Hi Thomas,
This is resolved